src/porepy/geometry/half_space.py

```diff
@@ -61,47 +54,41 @@ def half_space_interior_point(
 ) -> np.ndarray:
     """
     Find an interior point for the halfspaces.
+    
+    Note: 
+        We use linear programming to find one interior point for the half spaces.
+        Assume, num_planes halfspaces defined by
 
-    We use linear programming to find one interior point for the half spaces.
-    Assume, num_planes halfspaces defined by
-
-        aj*x1+bj*x2+cj*x3+dj<=0, j=1..num_planes.
+            aj*x1+bj*x2+cj*x3+dj<=0, j=1..num_planes.
 
-    Perform the following linear program:
+        Perform the following linear program:
 
-        max(x5) aj*x1+bj*x2+cj*x3+dj*x4+x5<=0, j=1..num_planes
+            max(x5) aj*x1+bj*x2+cj*x3+dj*x4+x5<=0, j=1..num_planes
 
-    Then, if [x1,x2,x3,x4,x5] is an optimal solution with x4>0 and x5>0 we get:
+        Then, if [x1,x2,x3,x4,x5] is an optimal solution with x4>0 and x5>0 we get:
 
-        aj*(x1/x4)+bj*(x2/x4)+cj*(x3/x4)+dj<=(-x5/x4) j=1..num_planes
+            aj*(x1/x4)+bj*(x2/x4)+cj*(x3/x4)+dj<=(-x5/x4) j=1..num_planes
 
-    and
-         (-x5/x4)<0,
+        and
```

Comment:
"and" is rendering in bold in the webpage. Please, fix it.

Reply:
I managed to fix this.